<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="cds-1809016" ignored="0" name="ZUP3Test" tests="3" failures="3" timestamp="2021-08-18T11:58:26 MSK" time="1.316" errors="0">
  <testcase name="testByUid" time="1.002" classname="cdsosobist.connectors.rest.testClient">
    <failure type="org.identityconnectors.framework.common.exceptions.ConnectorException" message="HTTP error 500 Internal server error : &amp;lt;!DOCTYPE HTML PUBLIC &amp;quot;-//W3C//DTD HTML 4.01//EN&amp;quot;&amp;gt;
&amp;lt;html&amp;gt;&amp;lt;head&amp;gt;&amp;lt;title&amp;gt;1C:Enterprise 8 application error&amp;lt;/title&amp;gt;&amp;lt;/head&amp;gt;&amp;lt;body&amp;gt;&amp;lt;h2&amp;gt;1C:Enterprise 8 application error:&amp;lt;/h2&amp;gt;Ошибка при разборе дескриптора виртуальных ресурсов&amp;lt;br&amp;gt;&amp;lt;b&amp;gt;by reason: &amp;lt;/b&amp;gt;&amp;lt;br&amp;gt;Файл не обнаружен &amp;apos;C:\inetpub\wwwroot\default.vrd&amp;apos;. 2(0x00000002): Не удается найти указанный файл. &amp;lt;/body&amp;gt;&amp;lt;/html&amp;gt;">
      <![CDATA[org.identityconnectors.framework.common.exceptions.ConnectorException: HTTP error 500 Internal server error : <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN">
<html><head><title>1C:Enterprise 8 application error</title></head><body><h2>1C:Enterprise 8 application error:</h2>Ошибка при разборе дескриптора виртуальных ресурсов<br><b>by reason: </b><br>Файл не обнаружен &apos;C:\inetpub\wwwroot\default.vrd&apos;. 2(0x00000002): Не удается найти указанный файл. </body></html>
at com.evolveum.polygon.rest.AbstractRestConnector.processResponseErrors(AbstractRestConnector.java:237)
at cdsosobist.connectors.rest.zup3Connector.processYResponseErrors(zup3Connector.java:825)
at cdsosobist.connectors.rest.zup3Connector.callORequest(zup3Connector.java:745)
at cdsosobist.connectors.rest.zup3Connector.handleIndividuals(zup3Connector.java:1371)
at cdsosobist.connectors.rest.zup3Connector.executeQuery(zup3Connector.java:928)
at cdsosobist.connectors.rest.testClient.testByUid(testClient.java:104)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testByUid -->
  <testcase name="testConn" time="0.007" classname="cdsosobist.connectors.rest.testClient">
    <failure type="org.identityconnectors.framework.common.exceptions.ConnectorException" message="HTTP error 500 Internal server error : &amp;lt;!DOCTYPE HTML PUBLIC &amp;quot;-//W3C//DTD HTML 4.01//EN&amp;quot;&amp;gt;
&amp;lt;html&amp;gt;&amp;lt;head&amp;gt;&amp;lt;title&amp;gt;1C:Enterprise 8 application error&amp;lt;/title&amp;gt;&amp;lt;/head&amp;gt;&amp;lt;body&amp;gt;&amp;lt;h2&amp;gt;1C:Enterprise 8 application error:&amp;lt;/h2&amp;gt;Ошибка при разборе дескриптора виртуальных ресурсов&amp;lt;br&amp;gt;&amp;lt;b&amp;gt;by reason: &amp;lt;/b&amp;gt;&amp;lt;br&amp;gt;Файл не обнаружен &amp;apos;C:\inetpub\wwwroot\default.vrd&amp;apos;. 2(0x00000002): Не удается найти указанный файл. &amp;lt;/body&amp;gt;&amp;lt;/html&amp;gt;">
      <![CDATA[org.identityconnectors.framework.common.exceptions.ConnectorException: HTTP error 500 Internal server error : <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN">
<html><head><title>1C:Enterprise 8 application error</title></head><body><h2>1C:Enterprise 8 application error:</h2>Ошибка при разборе дескриптора виртуальных ресурсов<br><b>by reason: </b><br>Файл не обнаружен &apos;C:\inetpub\wwwroot\default.vrd&apos;. 2(0x00000002): Не удается найти указанный файл. </body></html>
at com.evolveum.polygon.rest.AbstractRestConnector.processResponseErrors(AbstractRestConnector.java:237)
at cdsosobist.connectors.rest.zup3Connector.processYResponseErrors(zup3Connector.java:825)
at cdsosobist.connectors.rest.zup3Connector.callORequest(zup3Connector.java:745)
at cdsosobist.connectors.rest.zup3Connector.test(zup3Connector.java:131)
at cdsosobist.connectors.rest.testClient.testConn(testClient.java:88)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testConn -->
  <testcase name="testFindAll" time="0.008" classname="cdsosobist.connectors.rest.testClient">
    <failure type="org.identityconnectors.framework.common.exceptions.ConnectorException" message="HTTP error 500 Internal server error : &amp;lt;!DOCTYPE HTML PUBLIC &amp;quot;-//W3C//DTD HTML 4.01//EN&amp;quot;&amp;gt;
&amp;lt;html&amp;gt;&amp;lt;head&amp;gt;&amp;lt;title&amp;gt;1C:Enterprise 8 application error&amp;lt;/title&amp;gt;&amp;lt;/head&amp;gt;&amp;lt;body&amp;gt;&amp;lt;h2&amp;gt;1C:Enterprise 8 application error:&amp;lt;/h2&amp;gt;Ошибка при разборе дескриптора виртуальных ресурсов&amp;lt;br&amp;gt;&amp;lt;b&amp;gt;by reason: &amp;lt;/b&amp;gt;&amp;lt;br&amp;gt;Файл не обнаружен &amp;apos;C:\inetpub\wwwroot\default.vrd&amp;apos;. 2(0x00000002): Не удается найти указанный файл. &amp;lt;/body&amp;gt;&amp;lt;/html&amp;gt;">
      <![CDATA[org.identityconnectors.framework.common.exceptions.ConnectorException: HTTP error 500 Internal server error : <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN">
<html><head><title>1C:Enterprise 8 application error</title></head><body><h2>1C:Enterprise 8 application error:</h2>Ошибка при разборе дескриптора виртуальных ресурсов<br><b>by reason: </b><br>Файл не обнаружен &apos;C:\inetpub\wwwroot\default.vrd&apos;. 2(0x00000002): Не удается найти указанный файл. </body></html>
at com.evolveum.polygon.rest.AbstractRestConnector.processResponseErrors(AbstractRestConnector.java:237)
at cdsosobist.connectors.rest.zup3Connector.processYResponseErrors(zup3Connector.java:825)
at cdsosobist.connectors.rest.zup3Connector.callRequest(zup3Connector.java:767)
at cdsosobist.connectors.rest.zup3Connector.handleIndividuals(zup3Connector.java:1388)
at cdsosobist.connectors.rest.zup3Connector.executeQuery(zup3Connector.java:928)
at cdsosobist.connectors.rest.testClient.testFindAll(testClient.java:126)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- testFindAll -->
</testsuite> <!-- ZUP3Test -->
